apps/worker/database/models/core.py

```diff
@@ -307,11 +308,7 @@ def is_repo_covered_by_integration(self, repo: Repository) -> bool:
         return repo.service_id in self.repository_service_ids
 
     def is_configured(self) -> bool:
-        """Returns whether this installation is properly configured and can be used"""
-        if self.name == GITHUB_APP_INSTALLATION_DEFAULT_NAME:
-            # The default app is configured in the installation YAML
-            return True
-        return self.app_id is not None and self.pem_path is not None
+        return is_configured(self.app_id, self.pem_path)
```

Comment:
Maybe dumb q, but isn't this calling itself since you're rebinding `is_configured`?